src/cmd/build.js

```diff
@@ -124,11 +139,29 @@ async function defaultPackageCreator({
   const packageName = safeFileName(
     `${extensionName}-${manifestData.version}.zip`);
   const extensionPath = path.join(artifactsDir, packageName);
-  const stream = createWriteStream(extensionPath);
+
+  //Added 'wx' flags to avoid overwriting of existing package
```

Comment:
Nit, The comments should have a space between `//` and the comment, and ends with a '.'